### PR TITLE
Add suport to change threshold time.

### PR DESCRIPTION
**Pull request by bob1983**
This PR adds interface for threshold time.
We can change threshold time on our deploy.rb, like:

```
set :alert_threshold, 10
```

### CODE REVIEW

File: lib/capistrano/measure/log_reporter.rb [hunk 1]
```diff
@@ -4,11 +4,16 @@
 module Capistrano
   module Measure
     class LogReporter
-      ALERT_TRASHOLD = 60
-      WARNING_TRASHOLD = 30
+
+      attr_reader :alert_threshold, :warning_threshold
+
+      DEFALUT_ALERT_THRESHOLD = 60
+      DEFAULT_WARNING_THRESHOLD = 60
```
Comment (buehmann) [inline lib/capistrano/measure/log_reporter.rb]: Was meant to be 30 I guess.

Comment (AMekss) [inline lib/capistrano/measure/log_reporter.rb]: Yeah this should be 30 indeed

**Comment by AMekss** [comment]
@bob1983 PR it self looks good, could you fix `DEFAULT_WARNING_THRESHOLD` value, please?

**Comment by bob1983** [comment]
@AMekss 
Thank you for feedback 🙇 
I fixed it and rebased c5fcc8e .

**Pull request merged**